app/models/ansible/runner.rb

```diff
@@ -0,0 +1,27 @@
+module Ansible
+  class Runner
```

Comment:
Hm my only problem with this naming is that this is the pattern we use for workers and could be confusing, @Fryguy thoughts?

Reply:
also, I think I should move this under lib/